docs/development/Developers_Guide_de.md

```diff
@@ -0,0 +1,30 @@
+### Software-Anforderungen
+* Die folgenden Voraussetzungen müssen erfüllt sein:
+    - Oracle JDK / Open JDK 8 installieren
+    - Maven installieren (mit mvn -version prüfen, ob jdk 8 korrekt eingerichtet ist / falls nicht, JAVA_HOME ändern)
+    - Docker installieren
+
+### Lokales LDAP mit Docker einrichten
+* Checken Sie das folgende Repository aus https://github.com/cloudogu/docker-sample-ldap
+* Bauen Sie den Container `docker build -t usermgt/ldap .`
+* Starten Sie den Container `docker run --rm -p389:389 usermgt/ldap`.
+
+### Lokales LDAP mit CES einrichten
+* Binden Sie den ldap-Port an das Host-System (z.B. https://stackoverflow.com/questions/19335444/how-do-i-assign-a-port-mapping-to-an-existing-docker-container)
```

Comment:
this doesn't work for me because the folder from the docker container gets removed when i stop docker